package.json

```diff
@@ -25,8 +25,13 @@
   "license": "MIT",
   "scripts": {
     "android:dev-client": "IS_DEV=1 npx expo run:android",
-    "bootstrap:new_app": "yarn && node ./scripts/bootstrap.js",
-    "build:production:android": "yarn prepare:production && eas build --platform android --profile production",
+    "baca": "yarn build:baca-cli && node ./scripts/cli/build/scripts/cli",
+    "baca:bootstrap": "yarn build:baca-cli && yarn baca bootstrap",
+    "baca:generate": "yarn build:baca-cli && yarn baca generate",
+    "b": "yarn build:baca-cli && yarn baca b",
```

Comment:
```yarn build:baca-cli``` it seems, that this command is invoked to times. Could be removed from here and form next line, as it existn in ```yarn baca b```